08-CMSIS/Makefile

```diff
@@ -59,6 +59,21 @@ target:
 st-flash: $($(STM32F429_DEVICE)_TARGET)
 	st-flash --reset write $($(STM32F429_DEVICE)_TARGET_bin) 0x8000000
 
+
+oocd:
+	sudo openocd -f board/stm32f429discovery.cfg  \
```

Comment:
Don't hardcode board configurations here.

Reply:
Why? isn't the `st-flash` above hardcode the board configuration?

Reply:
No, we wish to support both QEMU-based stm32-p103 and stm32f429-discovery board.